### PR TITLE
Add comma separated multi labels facility

### LINKED ISSUE #32: Posibility to pass multiple values to an action
It would be great to be able to send multiple values in the same action, i.e:

`Derek add label: UX, enhancement, proposal` (`,` as separator, more natural)
or
`Derek add label: UX || enhancement || proposal` (`||` as separator)

I can work on this if the proposal is accepted

### LINKED ISSUE #110: Proposal: support multiple commands in one comment
After reading the docs I got the impression I can specify multiple commands, so I added the following comment (on https://github.com/moby/moby/pull/38634):
```
d3r3k add label: rebuild/windowsRS1
d3r3k add label: rebuild/janky
```
... and nothing happened, so I figured I should start with a capital D, and edited the comment accordingly:

```
D3r3k add label: rebuild/windowsRS1
D3r3k add label: rebuild/janky
```

## Expected Behaviour

I expected two labels to be added.

## Current Behaviour

It added a single label "rebuild/windowsRS1
D3r3k add label: rebuild/janky" (yes, with a new line (aka carriage return, aka `\n`, aka `%0D`) in between).

I tried to remove it by using something like
```
D3r3k remove label: rebuild/windowsRS1
D3r3k add label: rebuild/janky
```
but it didn't work. It might be a bug in Github rather than Derek though )

## Proposed solution

I see a few possible ways to handle it:

* break input into sections starting with `^Derek[[:space:]]` regexp, and handle each one separately;
* discard anything after the first line, if the input is not supposed to be multiline (such as in case of labels);
* keep the current behavior, but document it ("only one command at a time") and change the examples accordingly.

### PR DESCRIPTION
**Pull request by rgee0**
Signed-off-by: Richard Gee <richard@technologee.co.uk>

## Description
This change enables multiple labels to be applied through a single comment, by allowing for a comma separated list to be provided.  Also added plural aliases for the labels commands.

As the deletion of multiple labels is currently an iterative process, due to how the GitHub API is presented, an error could occur mid-flow when deleting and cause subsequent labels to not be applied. Mitigation against this is that labels are visually verifiable.

## Motivation and Context
Currently to add multiple labels a maintainer must add one comment for each label.  Members have found this confusing with some calling the command multiple times per comment.

 [x] I have raised an issue to propose this change (fixes #32, closes #110 )

## How Has This Been Tested?
Unit tests added.
Built Derek at `rgee0/derek:multiLabels`
Deployed locally
Tested on test repo - https://github.com/rgee0/Test/issues/7

image

image

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/derek/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by alexellis** [comment]
What's the limit in one command?

**Comment by alexellis** [comment]
screenshot 2019-02-17 at 16 39 43

I guess this is what I'd be concerned about - intentional or otherwise:

- Spam
- Abuse
- Exhausting rate limit

**Comment by alexellis** [comment]
What if we set a limit to something like 5 labels at once and have that configurable?

File: README.md [hunk 1]
```diff
@@ -97,12 +97,28 @@ Derek edit title: Question - does this work on Windows 10?
...
+and to remove:
+```
 Derek remove label: bug
 ```
 
+To address multiple labels through a single action use a comma separated list.
+
+To add multiple labels:
+```
+Derek add label: proposal, help wanted, skill/intermediate
```
Comment (alexellis) [inline README.md]: Can this work both with and without the space?

**Comment by alexellis** [comment]
Other than my comment here I think this is ready to merge. What do you think Richard?

File: README.md (hunk 1 above)
Comment (rgee0) [reply README.md]: Yes, its splits and then trims.  If there's no space then trim's output is the same as the input.

**Comment by rgee0** [comment]
> What if we set a limit to something like 5 labels at once and have that configurable?

And the action upon finding more?

**Comment by alexellis** [comment]
Set a reasonable value, then ignore anything over.

**Pull request closed without merging**